lib/rspec/core/configuration.rb

```diff
@@ -1712,7 +1712,8 @@ def extract_location(path)
 
         if match
           captures = match.captures
-          path, lines = captures[0], captures[1][1..-1].split(":").map { |n| n.to_i }
+          path = captures[0]
+          lines =  captures[1][1..-1].split(":").map(&:id)
```

Comment:
Looks like two issues here:

- double whitespace between `=` and `captures` in `lines =  captures[1]`
- the 2nd captures changed the block it needs to use `&:to_i` such as the following:

  ```ruby
  captures[1][1..-1].split(":").map(&:to_i)
  ```